### PR TITLE
JBPM-7112 Stuner Project showcase updating wildfly denpendency Errai

### PR DESCRIPTION
**Pull request by tiagodolphine**
Udpating wildfly dependency on the unpack maven goal, to the Errai patched wildfly.
The Stunner project showcase was not abbled to be stared from Intellij IDEA.
@romartin 
@mbarkley

### CODE REVIEW

File: kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/pom.xml [hunk 1]
```diff
@@ -1415,7 +1415,7 @@
             <configuration>
               <artifactItems>
                 <artifactItem>
-                  <groupId>org.wildfly</groupId>
+                  <groupId>org.jboss.errai</groupId>
                   <artifactId>wildfly-dist</artifactId>
                   <version>${version.org.wildfly.gwt.sdm}</version>
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/pom.xml]: that's wrong - it shoudl be  `<version>${version.org.jboss.errai.wildfly}</version>`

Comment (romartin) [reply kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/pom.xml]: @tiagodolphine right? please check it, I just compared with the standalone showcase...

Comment (mbarkley) [reply kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/pom.xml]: @romartin yeah `version.org.jboss.errai.wildfly` is the right version property.

Comment (tiagodolphine) [reply kie-wb-common-stunner/kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/pom.xml]: hey @mbarkley @romartin I've just updated the version. Thanks.

**Comment by romartin** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
@tiagodolphine Hey I was facing with the issue you told me about the `StunnerBPMNEntryPoint`.. please can you add the fix in this PR, so we can merge it all together? Here are the fixed files:
- The `beans.xml` for the `standalone` showcase -> [here](https://gist.github.com/romartin/08e7cde320e87e73588360fa0a017f85)
- The `beans.xml` for the `project` showcase -> [here](https://gist.github.com/romartin/6eee48bd9833e8059cba17cfd22adee0)

Thanks!!

**Comment by tiagodolphine** [comment]
@romartin updated with the fixes on beans.xml

**Comment by tiagodolphine** [comment]
Jenkins execute full downstream build

**Pull request merged**